JSDemos/Demos/Tabs/Overview/Angular/app/app.component.css

```diff
@@ -60,10 +59,10 @@
 }
 
 ::ng-deep .dx-viewport:not(.dx-theme-generic) .dx-tabs-horizontal {
-  border-bottom: 1px solid rgb(225, 225, 225, 0.4);
+  border-block-end: 1px solid rgb(225, 225, 225, 0.4);
```

Comment:
do we really have to use logical prop here?

Reply:
It doesn't matter.